frontend/src/components/institute/MoveSchoolPage.vue

```diff
@@ -0,0 +1,603 @@
+    this.$store.dispatch('institute/getAllActiveSchoolNeighborhoodLearningCodes');
+    this.$store.dispatch('institute/getAllActiveProvinceCodes');
+    this.$store.dispatch('institute/getAllActiveCountryCodes');
+    this.$store.dispatch('institute/getSchoolCategoryFacilityTypesMap');
+    this.getActiveDistrictDropDownItems();
+    this.getActiveAuthorityDropDownItems(); 
+  },
+  methods: {
+    getActiveDistrictDropDownItems() {
+      ApiService.getActiveDistricts().then((response) => {
+        for(const district of response.data){
+          let districtItem = {
+            districtNumberName: `${district.districtNumber} - ${district.name}`,
+            districtId: district.districtId,
+          };
+          this.activeDistricts.push(districtItem);
+        }
+        this.activeDistricts = this.sortByNameValue(this.activeDistricts, 'districtNumberName');
+      }).catch(error => {
+        console.error(error);
+        this.setFailureAlert('An error occurred while getting active districts. Please try again later.');
+      });
+    },
+    getActiveAuthorityDropDownItems(){
+      ApiService.getActiveAuthorities().then((response) => {
+        for(const authority of response.data){
+          let authorityItem = {
+            authorityNumber: +authority.authorityNumber,
+            authorityCodeName: `${authority.authorityNumber} - ${authority.name}`,
+            authorityID: authority.authorityID,
+          };
+          this.activeAuthorities.push(authorityItem);
+        }
+        this.activeAuthorities = this.activeAuthorities.sort(function(a, b){return a.authorityNumber-b.authorityNumber;});
+      }).catch(error => {
+        console.error(error);
+        this.setFailureAlert('An error occurred while getting active authorities. Please try again later.');
+      });
+    },
+    enableOrDisableFacilityType(facilityTypes){
+      this.isFacilityTypeDisabled = facilityTypes && facilityTypes.length === 1;
+      if(this.isFacilityTypeDisabled){
+        this.moveSchoolObject.facilityTypeCode = facilityTypes[0].facilityTypeCode;
+      }else{
+        this.moveSchoolObject.facilityTypeCode = null;
+      }
+  
+      this.fireFormValidate();
+    },
+    isIndependentOnlyUser() {
+      return this.SCHOOL_INDEPENDENT_ADMIN_ROLE;
+    },
+    authorityRule(value) {
+      if (this.moveSchoolObject.schoolCategoryCode && this.requiredAuthoritySchoolCategories.includes(this.moveSchoolObject.schoolCategoryCode) && !value) {
+        return 'Required';
+      }
+    },
+    saveMoveSchoolDate(date) {
+      this.$refs.moveDateFilter.save(date);
+    },
+    closeMoveSchoolPage() {
+      this.resetForm();
+      this.$emit('moveSchool:closeMoveSchoolPage');
+    },
+    moveSchool() {
+      this.processing = true;
+      ApiService.apiAxios.post(`${Routes.institute.SCHOOL_MOVE_URL}`, this.moveSchoolObject)
+        .then(() => {
+          this.setSuccessAlert('Success! The school has been created.');
```

Comment:
should this be school has been moved?

Reply:
No, I have updated the message to something else